psw/urts/linux/sig_handler.cpp

```diff
@@ -353,10 +353,11 @@ static int sgx_urts_vdso_handler(long rdi, long rsi, long rdx, long ursp, long r
                 return 0;
             }
             //move the ocall return result to rsi and set rdi to ECMD_ORET for ocall return to trts
-            __asm__ __volatile__("mov %%rax, %%rsi\n"
-                    "mov %0, %%rdi\n"
+            __asm__ __volatile__("mov $0, %%rsi\n"
```

Comment:
One clarificationn question for my own understanding: 
Don't the first two mov instruction have the same effect?

Reply:
Hi Anand, the first line is $0, the second line is %0, they looks almost the same.

Reply:
To avoid misleading, perhaps it is better to use `xor` instead of `mov` to clear the register, such as `xor  %%rsi, %%rsi`?

Reply:
Thanks for pointing it out @JunjunGu1 . Now, it is clear.